motoko/ios-notifications/dapp-demo/dfx.json

```diff
@@ -0,0 +1,34 @@
+{
+  "canisters": {
+    "ios_notifications_api": {
+      "main": "src/api/main.mo",
+      "type": "motoko"
+    },
+    "internet_identity": {
+      "type": "custom",
+      "candid": "internet_identity.did",
+      "wasm": "internet_identity.wasm",
+      "build": "bash -c 'test -f internet_identity.wasm || curl -sSL https://github.com/dfinity/internet-identity/releases/latest/download/internet_identity_dev.wasm -o internet_identity.wasm; test -f internet_identity.did || curl -sSL https://raw.githubusercontent.com/dfinity/internet-identity/main/src/internet_identity/internet_identity.did -o internet_identity.did'",
```

Comment:
It's no longer necessary to download manually. You can use this: https://github.com/dfinity/internet-identity/blob/main/demos/using-dev-build/dfx.json